src/providers/Wagmi/updater.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+
+// wagmi
+import { useAccount, useConnect, useDisconnect } from 'wagmi';
+
+// Redux
+import { useAppDispatch, useAppSelector } from '../../store';
+import { Store } from '../../types/index';
+import { web3Actions } from '../../store/slices/web3';
+
+export default function WagmiProvider() {
```

Comment:
maybe we should call this WagmiUpdater as there is already a WagmiProvider

Reply:
Good point, changed